cpp/src/barretenberg/stdlib/recursion/transcript/transcript.hpp

```diff
@@ -267,9 +282,25 @@ template <typename Composer> class Transcript {
 
```

Comment:
This block of code only executes for `num_challenges > 2`, which (currently) only happens in the `nu` round when we need to generate short scalars. In this case, we generate 32-byte challenges and split them in half to get the relevant challenges.